### PR TITLE
[zh] Translate migrate from PSP

### PR DESCRIPTION
**Pull request by tengqm**
/language zh

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  - Update all ClusterRoles authorizing use of those mutating PSPs to also authorize use of the
+    non-mutating variant.
+  - Watch for Pods using the mutating PSPs and work with code owners to migrate to valid,
+    non-mutating resources.
+  - Delete mutating PSPs.
+-->
+-- **如果你的集群中配置了变更式的 PodSecurityPolicy，将它们删除。**
+   - 复制所有变更式 PSP 复制到非变更式版本中。
+   - 更新所有授权使用那些变更式 PSP 的 ClusterRole，使之也能为非变更式版本鉴权。
+   - 检视使用了变更式 PSP 的 Pods，与拥有该代码的人一起将其迁移到合法的、非变更式的资源。
```
Comment (howieyuen) [inline content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md:55]:
```suggestion
   - 检视使用了变更式 PSP 的 Pod，与拥有该代码的人一起将其迁移到合法的、非变更式的资源。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md:55]: OK

File: content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  they are in compliance with the selected profiles.
+- Apply the profiles in `enforce` mode.
+- Stop including `PodSecurityPolicy` in the `--enable-admission-plugins` flag.
+-->
+- **在 `warn` 和 `audit` 模式下应用所选的策略。**
+  这样做会让你了解 Pod 会如何对新的策略作出反应，同时不会破坏现有负载。
+  反复调试你的[Pod 配置](/zh/docs/concepts/security/pod-security-admission#configuring-pods)
+  直到它们与所选的策略匹配。
+- 用 `enforce` 模式下应用策略。
+- 在 `--eanble-admission-plugins` 标志中去除 `PodSecurityPolicy`。
```
Comment (sftim) [inline content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md:92]: nit:
```suggestion
- 在 `--enable-admission-plugins` 标志中去除 `PodSecurityPolicy`。
```

Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/migrate-from-psp.md]: Thanks. Fixed.

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**